src/main/java/nl/mrwouter/minetopiafarms/events/TreeFarmer.java

```diff
@@ -21,12 +21,10 @@ public void onBreak(BlockBreakEvent e) {
 		Player p = e.getPlayer();
 
 		if (e.getBlock().getType().toString().contains("LOG") && CustomFlags.hasFlag(p, e.getBlock().getLocation())) {
-			if (Main.getPlugin().getConfig().getBoolean("PrefixEnabled")) {
-				if (!PlayerManager.getOnlinePlayer(e.getPlayer().getUniqueId()).getPrefix().equalsIgnoreCase("Houthakker")) {
-					e.getPlayer().sendMessage(Main.getMessage("BeroepNodig").replaceAll("<Beroep>", "houthakker"));
-					e.setCancelled(true);
-					return;
-				}
+			if (Main.getPlugin().getConfig().getBoolean("PrefixEnabled") && (!PlayerManager.getOnlinePlayer(e.getPlayer().getUniqueId()).getPrefix().equalsIgnoreCase("Houthakker"))) {
```

Comment:
Waarom dubbele haakjes om de prefix check?

Reply:
Waarom niet? Meer is beter kappa